sdk/python/pulumi_random/random_id.py

```diff
@@ -59,6 +59,26 @@ def __init__(__self__, resource_name, opts=None, byte_length=None, keepers=None,
         unique names during the brief period where both the old and new resources
         exist concurrently.
 
+        ## Example Usage
+
+
+
+        ```python
+        import pulumi
+        import pulumi_aws as aws
+        import pulumi_random as random
+
+        server_random_id = random.RandomId("serverRandomId",
+            byte_length=8,
+            keepers={
+                "ami_id": var["ami_id"],
```

Comment:
Note to self: `tf2pulumi` should synthesize appropriately-typed definitions for missing variables s.t. we see `ami_id` here instead of `var["ami_id"]`.